bliss._.js

```diff
@@ -58,10 +58,12 @@ Object.defineProperty(Array.prototype, _, {
 if (self.EventTarget && "addEventListener" in EventTarget.prototype) {
 	var addEventListener = EventTarget.prototype.addEventListener,
 	    removeEventListener = EventTarget.prototype.removeEventListener,
-	    equal = function(callback, capture, l){
-	    	return l.callback === callback && l.capture == capture;
+	    equal = function(callback, capture, l) {
+			return l.callback === callback && l.capture == capture;
```

Comment:
They have nothing to do with assignments. Smart tabs are the practice of using tabs for indentation and spaces for alignment. Though I guess in this case it could go either way.